apps/webapp/app/routes/api.v1.runs.$runId.logs.ts

```diff
@@ -1,8 +1,8 @@
 import type { Organization, RuntimeEnvironment } from "@trigger.dev/database";
 import type { ActionArgs } from "@remix-run/server-runtime";
 import { json } from "@remix-run/server-runtime";
-import type { LogMessage } from "@trigger.dev/internal";
-import { LogMessageSchema } from "@trigger.dev/internal";
+import type { LogMessage } from "@trigger.dev/core";
+import { LogMessageSchema } from "@trigger.dev/core";
```

Comment:
This should be moved into a server file to make sure core isn't being included in the browser bundle